qiskit_addon_sqd/fermion.py

```diff
@@ -98,8 +99,9 @@ def solve_fermion(
             halves of the bitstrings should be kept separate. If ``False``, CI strings
             from the left and right halves of the bitstrings are combined into a single
             set of unique configurations and used for both the alpha and beta subspaces.
-        spin_sq: Target value for the total spin squared for the ground state.
+        spin_sq: Target value for the total spin squared for the ground state, :math:`S^2 = s(s + 1)`.
             If ``None``, no spin will be imposed.
+        shift: Level shift for states which have different spin. :math:`(H + shift * S^2)|ψ> = E|ψ>`
```

Comment:
maybe this equation should go up in the ``spin_sq`` description? Maybe even in top-level description of this function?